packages/interface/src/features/proposals/components/MetadataForm.tsx

```diff
@@ -72,7 +74,7 @@ export const MetadataForm = ({ pollId }: IMetadataFormProps): JSX.Element => {
   const { error: createError } = create;
 
   return (
-    <div className="dark:border-lighterBlack rounded-lg border border-gray-200 p-4">
+    <div className="dark:border-lighterBlack flex w-full max-w-[824px] flex-col gap-10 rounded-lg border border-gray-200 p-5">
```

Comment:
I think maybe it's better to just keep `w-[824px]` since then the form width won't be dynamic